.travis.yml

```diff
@@ -2,16 +2,14 @@ language: d
 sudo: false
 
 d:
+  - dmd-nightly
+  - dmd-beta
   - dmd
+  - ldc-beta
   - ldc
```

Comment:
Btw Travis crons are highly useful: https://blog.travis-ci.com/2016-12-06-the-crons-are-here